stacks/op-scim-bridge/deploy.sh

```diff
@@ -2,19 +2,18 @@
 
 set -e
 
-# create namespace
-cat <<EOF | kubectl apply -f -
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: op-scim-bridge
-EOF
+helm repo add op-scim-bridge 'https://raw.githubusercontent.com/1Password/op-scim-helm/main/'
+helm repo update
 
-# set kubectl namespace
-kubectl config set-context --current --namespace=op-scim-bridge
+STACK="op-scim"
+CHART="op-scim-bridge/op-scim"
+CHART_VERSION="1.4.4"
+NAMESPACE="op-scim-bridge"
 
-# deploy op-scim-bridge
-kubectl apply -f https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/op-scim-bridge/yaml/op-scim-bridge.yaml
+helm upgrade "$STACK" "$CHART" \
+  --install \
+  --create-namespace \
+  --namespace "$NAMESPACE" \
+  --version "$CHART_VERSION" \
+  --wait
```

Comment:
Lets remove `--wait`
```
helm upgrade "$STACK" "$CHART" \
  --install \
  --create-namespace \
  --namespace "$NAMESPACE" \
  --version "$CHART_VERSION"
```

Reply:
Hi Paul! Could you please explain why we should remove the wait flag?

Reply:
I should've also mentioned to add `--atomic`. The `atomic` flag "`if set, upgrade process rolls back changes made in case of failed upgrade. The --wait flag will be set automatically if --atomic is used`"
```
helm upgrade "$STACK" "$CHART" \
  --atomic \
  --install \
  --create-namespace \
  --namespace "$NAMESPACE" \
  --version "$CHART_VERSION"
```

Reply:
Thank you for explaining. We are in the process of adding this to our PR. Do you have any documentation for how to work with the Vendor portal once this PR merges? Thanks!

Reply:
@paulanunda This is ready for final review. Thank you!

Reply:
LGTM. We don't have public documentation around onboarding into the Vendor Portal. I'll make the listing update on your behalf, and work w/ our product team to get that documentation drafted and shared. That way, y'all can self service next time!

Reply:
Perfect! Thank you!